features/support/formatter.rb

```diff
@@ -107,6 +107,7 @@ def before_step(step)
 
       #
 
+      # rubocop:disable Metrics/ParameterLists
```

Comment:
:100: Increase the size in Rubocop.  6 is a good number if you ask me, if it's over size I don't know what we should increase it to.  I vote increase it mostly because Rubocop does count kwd's, so since it's already impacting us without them, it'll impact us more with them if we ever do use them (which we do in a few places.)